package.json

```diff
@@ -8,11 +8,6 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
```

Comment:
Ah here's the culprit! `npm start` is still pointing to the react-native bundler instead of webpack. Have a look in the package.json of that project and test it out with webpack _for realsies_